plugins/BEdita/Core/config/Migrations/20170607114052_AddIdVatNumbers.php

```diff
@@ -0,0 +1,55 @@
+<?php
+use Migrations\AbstractMigration;
+
+class AddIdVatNumbers extends AbstractMigration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function up()
+    {
+        $this->table('profiles')
+            ->addColumn('national_id_number', 'string', [
+                'comment' => '',
+                'null' => true,
+                'default' => null,
+                'limit' => 32,
+            ])
+            ->addColumn('vat_number', 'string', [
+                'comment' => '',
```

Comment:
Comments for these two fields are missing. However column name is quite self-explanatory.